store/tikv/client_batch.go

```diff
@@ -474,12 +477,16 @@ func sendBatchRequest(
 	}
 	ctx1, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
+retry:
 	select {
 	case batchConn.batchCommandsCh <- entry:
 	case <-ctx1.Done():
 		logutil.BgLogger().Warn("send request is cancelled",
 			zap.String("to", addr), zap.String("cause", ctx1.Err().Error()))
 		return nil, errors.Trace(ctx1.Err())
+	case msg := <-c.eventCh:
+		handleEventMSG(c, msg)
+		goto retry
```

Comment:
Why we need a `retry`?

Reply:
If there is no retry, how to handle this entry?